lib/bundler/settings.rb

```diff
@@ -222,6 +222,10 @@ def app_cache_path
       end
     end
 
+    def global_cache_path
+      "~/.bundle/cache"
```

Comment:
`~` doesn't work on Windows, so well need to use whatever Bundler currently uses to find the global settings file.